ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                                <note/>
+                            </msItem>
+                            <msItem xml:id="ms_i15">
+                                <locus from="152va" to="159va"/>
+                                <title xml:lang="en" type="complete" ref="LIT7292MartyrMa">The
+                                    Martyrdom of Mercurius (recension a)</title>
+                                <incipit type="title inscriptio" xml:lang="gez"><hi rend="rubric"
+                                        >በስመ፡ አብ፡ ወወልድ፡</hi> ወመንፈስ፡ ቅዱስ፡ አሓዱ፡<hi rend="rubric">
+                                        አምላክ፡ ዝንቱ፡ ነ</hi>ገር፡ ዘቅዱስ፡ መርቆርዮስ፡ <hi rend="rubric">ጸሎቱ፡
+                                        ወበረከቱ። </hi></incipit>
+                                <incipit xml:lang="gez"> ወኮነ፡ በመዋዕለ፡ ዳክዮስ፡ ንጉሠ፡ ሮም፡ ወተወልደ፡ ሕፃን፡
+                                    ፒሎባዴር፡ ዘበትርጓሜሁ፡ መርቆርዮስ። ወስመ፡ አቡሁ፡ አሬስ፡ ወስመ፡ እምሔዉ፡ ሲዴሮስ። ወክልኤሆሙ፡
+                                    ነዓውያነ፡ አራዊት፡ </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i16">
+                                <locus from="160r" to="162va"/>
+                                <title xml:lang="en" type="complete" ref="LIT7294MartyrMc">Vita of
+                                    Mercurius (recension c)</title>
+                                <incipit type="title inscriptio" xml:lang="gez"><hi rend="rubric"
+                                        >በስመ፡ አብ፡ ወወልድ፡ ወመ</hi>ንፈስ፡ ቅዱስ፡ ገድል፡ ዘቅ<hi rend="rubric"
+                                        >ዱስ፡ ሰማዕት፡ ወኀያል፡ ወጽ</hi>ኑዕ፡ መርቆርዮስ፡ ዘፈጸ<hi rend="rubric">መ፡
+                                        ገድሉ፡ በወርኀ፡ ኅዳር፡ </hi>አመ፡ <hi rend="rubric">፳</hi>ወ<hi
+                                        rend="rubric">፭፡</hi> ጸሎቱ፡ ወበረከቱ፡ ተሀሉ፡ ምስሌነ፡ አሜን።
+                                    ።</incipit>
+                                <incipit xml:lang="gez">ወኮነ፡ በመዋዕለ፡ ዳኪዮስ፡ ንጉሥ፡ መምዕላይ፡ ዘይሰግድ፡ ለጣዖት።
+                                    ወሀሎ፡ ብእሲ፡ ዘስሙ፡ ቢስጦስ፡ ዘአምነ፡ በክርስቶስ፡ እምኃበ፡ ዐበይተ፡ ንጉሥ፡ ወሀሎ፡ ብእሲ፡
+                                    ዘስሙ፡ ፒሉፓዴር፡ ወምሁረ፡ ጥበብ፡ ወኵሎ፡ ፈጸመ፡ ወምህሮ፡ ኲናት፡ ወተፅዕኖ፡ አፍራስ፡ ወንድፈተ፡
+                                    ቀሥት፡ ወኵሎ፡ ሕሊና፡ ልቡ፡ ወኮነ፡ የአምን፡ በክርስቶስ፡ በኅቡእ። ወኮነ፡ ያነብብ፡ መጻሕፍተ፡
+                                    ላዕለ፡ ወልዱ፡ ፒሉፓዴር፡ ወይኤዝዝ፡ ወያጸንዕ፡ ስሞ፡ ለክርስቶስ፡ ወአምነ፡ ቦቱ። </incipit>
+                                <explicit xml:lang="gez">ወመጠወ፡ ክሳዶ፡ ወመተርዎ፡ ርእሶ፡ ወተፈጸመ፡ ስምዑ፡ አመ፡ ፳ወ፭
+                                    ለኅዳር፡ በሰላመ፡ እግዚአብሔር፡ አሜን። ወገብረ፡ መንክረ፡ ቅዱስ፡ መርቆርዮስ፡ ጸሎቱ፡ ወበረከቱ፡
+                                    ተሀሉ፡ ምስሌነ፡ አሜን።</explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i17">
+                                <locus from="162va" to="162vb"/>
+                                <title xml:lang="en" type="complete" ref="LIT7303TammeraBishops"
+                                    >Miracle of Mercurius: Death of the Emperor Julian the Apostate
```

Comment:
broken line

Reply:
> broken line

I wonder whether it actually affect the result because it the automatic formatting breaks lines.